main/utils/errors.js

```diff
@@ -1,34 +1,161 @@
 'use strict';
 
-const {dialog, clipboard} = require('electron');
+const os = require('os');
+const {clipboard, shell, app} = require('electron');
 const ensureError = require('ensure-error');
 const cleanStack = require('clean-stack');
+const isOnline = require('is-online');
+const {openNewGitHubIssue} = require('electron-util');
+const got = require('got');
+const delay = require('delay');
 
-const showError = async (error, {title, reportToSentry} = {}) => {
+const {showDialog} = require('../dialog');
+
+const MAX_RETRIES = 20;
+
+const getSentryIssue = async (eventId, tries = 0) => {
+  if (tries > MAX_RETRIES) {
+    return;
+  }
+
+  try {
+    const {body} = await got.get(`https://kap-sentry-tracker.vercel.app/api/event/${eventId}`, {json: true});
```

Comment:
For future readers of this code, I would add a short comment about what this endpoint does.